backend/app/Services/Warehouse/MaterialStockReconciler.php

```
@@ -0,0 +1,70 @@
+ *
+ * The difference is booked as an `adjustment` in the stock_movements ledger
+ * rather than written silently, so the jump has an audit trail.
+ */
+class MaterialStockReconciler
+{
+    public function __construct(private StockMovementService $stockMovements) {}
+
+    public function reconcile(int $materialId): void
+    {
+        DB::transaction(function () use ($materialId) {
+            $material = Material::find($materialId);
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\Material::find().


(staticMethod.notFound)